lib/src/controls/yaru_popup_menu_button.dart

```diff
@@ -9,86 +9,87 @@ class YaruPopupMenuButton<T> extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return DefaultTextStyle(
-      style: Theme.of(context).textTheme.bodyMedium ??
-          TextStyle(
-            fontSize: 20,
-            color: Theme.of(context).colorScheme.onSurface,
-          ),
-      child: ClipRRect(
-        borderRadius: BorderRadius.circular(kYaruButtonRadius),
-        child: Material(
-          color: Colors.transparent,
-          child: PopupMenuButton(
-            padding: EdgeInsets.zero,
-            initialValue: initialValue,
-            onSelected: onSelected,
-            onCanceled: onCanceled,
-            tooltip: tooltip,
-            itemBuilder: (context) {
-              return items;
-            },
-            child: YaruPopupDecoration(
-              child: child,
-            ),
+    return ClipRRect(
+      borderRadius: BorderRadius.circular(kYaruButtonRadius),
+      child: Material(
+        color: Colors.transparent,
+        child: PopupMenuButton(
+          padding: EdgeInsets.zero,
+          initialValue: initialValue,
+          onSelected: onSelected,
+          onCanceled: onCanceled,
+          tooltip: tooltip,
+          itemBuilder: itemBuilder,
+          child: _YaruPopupDecoration(
+            child: child,
+            childPadding: childPadding,
           ),
         ),
       ),
     );
   }
 }
 
-class YaruPopupDecoration extends StatelessWidget {
-  const YaruPopupDecoration({
+class _YaruPopupDecoration extends StatelessWidget {
+  const _YaruPopupDecoration({
+    // ignore: unused_element
     super.key,
     required this.child,
-    this.childPadding = const EdgeInsets.symmetric(horizontal: 5),
+    required this.childPadding,
   });
 
   final Widget child;
   final EdgeInsets childPadding;
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-        borderRadius: BorderRadius.circular(kYaruButtonRadius),
-        border: Border.all(color: Theme.of(context).dividerColor),
+    return DefaultTextStyle(
+      style: TextStyle(
+        color: Theme.of(context).colorScheme.onSurface,
+        fontWeight: FontWeight.w500,
       ),
-      child: Padding(
-        padding: childPadding,
-        child: Row(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            Padding(
-              padding: childPadding,
-              child: child,
-            ),
-            const SizedBox(
-              height: 40,
-              child: Icon(
-                YaruIcons.pan_down,
-                size: 20,
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(kYaruButtonRadius),
+          border: Border.all(color: Theme.of(context).dividerColor),
+        ),
+        child: Padding(
+          padding: childPadding,
+          child: Row(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              Padding(
+                padding: childPadding,
```

Comment:
the same padding is inserted again so you actually get double what you specify?

Reply:
No

Reply:
![image](https://user-images.githubusercontent.com/140617/197189803-0a6dd5af-dd6d-4671-8188-98b333b219db.png)

Reply:
![image](https://user-images.githubusercontent.com/140617/197190243-dc2f30fb-604e-42f3-8128-00ae0137dc0b.png)

Reply:
perhaps the second padding should be just a sized box to get some spacing between the child and the dropdown icon?